spotinst/elastigroup_aws_suspend_processes/consts.go

```diff
@@ -0,0 +1,9 @@
+package elastigroup_aws_suspend_processes
+
+import "github.com/terraform-providers/terraform-provider-spotinst/spotinst/commons"
+
+const (
+	GroupID     commons.FieldName = "group_id"
+	Suspensions commons.FieldName = "suspension"
```

Comment:
The name of the const (`Suspensions`) is plural while the value (`suspension `) is singular.

```suggestion
	Suspension commons.FieldName = "suspension"
```